python/eight_mile/pytorch/layers.py

```diff
@@ -137,7 +137,7 @@ def rnn_bi_hidden(output, output_state):
         output_state = tuple(_cat_dir(h) for h in output_state)
     else:
         output_state = _cat_dir(output_state)
-    return output, output_state
+    return output_state
```

Comment:
this is going away

Reply:
What should we do about that? Should this wait and be updated once you 2.0 branch is merged in? Or should I make test and make this PR against that branch?